### ISSUE TITLE
Is it possible to get updated 5010 based 270/271 mapping files?

### ISSUE THREAD

**Issue by bowden-mike** [imsweb/x12-parser#47]
Where do these mapping files come from?  Do they already exist in some form or do you have to create them from scratch?
Based on the error returned from the X12Reader, it looks like we need something like:         

ANSI270_5010_X279("mapping/270.5010.X279.A1.xml"),
ANSI271_5010_X279("mapping/271.5010.X279.A1.xml");

Is there anything I can do to help this process along?

Thanks.

**Comment by angelaszek** [comment]
They might already exist in some form in another open source project somewhere. I did a quick look but didn't find any but I may have missed it. Otherwise, they need to be created from scratch using the published specs for these standards.

**Comment by ctmay4** [comment]
The files this project uses originated in a Python open source project.  See https://github.com/azoner/pyx12.  

Their files are located here:

https://github.com/azoner/pyx12/tree/master/pyx12/map

**Comment by bowden-mike** [comment]
Thanks Angela & Chuck.  Given that 5010 was implemented back in 2009, I'm not holding my breath.  I've resigned myself to doing the part I need (i.e. Medicare doesn't and will never require eligibility for auto insurance).  I can pass on the piece that I map, but given how the other specs are complete, I would hate to pollute the purity of what you have in place with a partial spec.

All the same, thanks for the response/confirmation.
